benchmark/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>ldaptive-benchmark</artifactId>
```

Comment:
Did you generate this artifact/module using the Maven archtype as recommended in the docs, https://openjdk.java.net/projects/code-tools/jmh/?

Reply:
I believe I did, but then immediately hacked it up to look like the existing pom files. Did I remove something important?

Reply:
Not that I can see. I was mostly just curious.